core/src/main/java/io/ddf/content/ViewHandler.java

```diff
@@ -128,7 +128,7 @@ public DDF removeColumns(List<String> columnNames) throws DDFException {
     }
     List<String> columns = this.getDDF().getColumnNames();
```

Comment:
`ViewHandler` was changed by @Huandao0812, I didn't change anything there:
https://github.com/ddf-project/DDF/commits/feature/immutable/core/src/main/java/io/ddf/content/ViewHandler.java

I noticed something crazy with removeColumns() on this branch too. Will take a look later, unless @Huandao0812 wanna look into this first.